bvm/ballerina-runtime/README.md

```diff
@@ -0,0 +1,241 @@
+| io.ballerina.runtime.api.Future          | This will contain the future value once we call the Ballerina method from API asynchronously.                                                                                                                                                                                                                                                                                                    |
+| io.ballerina.runtime.api.Module          | Represent Java runtime module.                                                                                                                                                                                                                                                                                                                                                                   |
+| io.ballerina.runtime.api.PredefinedTypes | Contains predefined types.                                                                                                                                                                                                                                                                                                                                                                       |
+| io.ballerina.runtime.api.Runtime         | An instance of the current runtime can be obtained through an `Environment` instance. This will contain APIs to call Ballerina object methods asynchronously.                                                                                                                                                                                                                                    |
+| io.ballerina.runtime.api.TypeTags        | Contains runtime type tags.                                                                                                                                                                                                                                                                                                                                                                      |
+
+## Create a Ballerina value
+
+The **io.ballerina.runtime.api.creators.ValueCreator**  provides APIs to create Ballerina value instances. A Ballerina value can be created using one of the methods provided in the ValueCreator class as per the requirements.
+
+For example, an array value can be created as follows,
+
+```
+BArray array = ValueCreator.createArrayValue(arrayType);
+```
+
+## Create a Ballerina type
+
+`io.ballerina.runtime.api.creators.TypeCreator` class provides APIs to create Ballerina type instances. There are multiple methods available to create Ballerina type instances based on the requirements.
+
+For example, a string type array can be created as follows.
```

Comment:
Should we mention the Record/Object type creation as a note? That we can't use them for value creation.